antispam_bee.php

```diff
@@ -2391,6 +2408,21 @@ public static function delete_spam_reason_by_comment( $comment_id ) {
 		);
 	}
 
+	/**
+	 * Updates the Antispam Bee reason for manual transitions
+	 *
+	 * @since   2.9.2
+	 * @param  [int] $comment Comment Object or comment ID.
+	 */
+	public static function update_antispam_bee_reason( $comment ) {
+		if ( ! is_object( $comment ) ) {
+			$comment = get_comment( $comment );
+		}
+		$id = $comment->comment_ID;
+		update_comment_meta( $id, 'antispam_bee_reason', 'manually' );
```

Comment:
I would just
`update_comment_meta( $comment->comment_ID, 'antispam_bee_reason', 'manually' );`

Reply:
Yes, this could be written shorter in just one line then.